web/styles/left_sidebar.css

```diff
@@ -224,14 +219,115 @@ li.show-more-topics {
     }
 }
 
-.narrows_panel {
-    & li a {
-        margin-top: 1px;
+#views-label-container {
+    cursor: pointer;
+    /* Padding of #private_messages_section_header. */
+    padding: 0 10px 1px 6px;
+    /* The margin-left of .private_messages_container. */
+    margin-left: 6px;
+    /* Space between #views-label-container and #left-sidebar-navigation-list in collapsed state. */
+    margin-right: 10px;
+    margin-top: 3px;
+    outline: none;
+}
 
-        &:hover {
+#left-sidebar-navigation-area {
+    & li.top_left_row {
```

Comment:
Once the rows are all gridded, which I'm working on, this CSS will need to be revisited. Just another thing to note for future improvements.